src/components/talks/AnimatedSlide/styled.ts

```diff
@@ -0,0 +1,142 @@
+import styled, { css } from 'styled-components';
+import media from '../../../styles/media';
+import { gridSize, sizes } from '../../../styles/variables';
+
+export const Container = styled.figure`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  margin: 48px -8px -8px -8px;
+
+  font-family: 'Merriweather', Georgia, serif;
+
+  > * {
+    margin: 8px;
+  }
+`;
+
+interface ImageWrapperProps {
+  useImageBorder: boolean;
+  isSectionHeader: boolean;
+}
+
+// prettier-ignore
+export const ImageWrapper = styled.a`
+  position: relative;
+  display: block;
+  width: 100%;
+  max-width: 500px;
+  border: thin solid
+    ${(props: ImageWrapperProps) =>
+      props.useImageBorder ? '#ccc' : 'transparent'};
+
+  ${(props: ImageWrapperProps) => props.isSectionHeader && css`
+    ${media.notSmall`
+      max-width: 700px;
+    `}
+
+    ${media.small`
+      margin: 0 -${sizes.contentPadding}px;
+      width: calc(100% + ${sizes.contentPadding}px * 2);
+      max-width: none;
+    `}
+  `}
+
+  &::before {
+    content: '#';
+
+    position: absolute;
+    top: -30px;
+    left: -135px;
+
+    /* Increase the area to allow moving the mouse from the image to the element */
+    width: 180px;
+    height: 120px;
+    text-align: center;
+    line-height: 120px;
+
+    color: #ccc;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 60px;
+    font-weight: bold;
+
+    /* Hide & animate the element */
+    opacity: 0;
+    transform: scale(0.8);
+    pointer-events: none;
+    transition: all 0.15s ease-out;
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    &::before {
+      opacity: 1;
+      transform: scale(1);
+      pointer-events: auto;
+      transition: none;
+    }
+  }
+
+  > * {
+    width: 100%;
+  }
+`;
```

Comment:
Is this something we could/should share with `Slide/styled.ts`? Or maybe this whole `AnimatedSlide` component should extend a `Slide`, dunno? (Curious what you think!)

Thinking of this simply because
- this copies a lot of styles from the `Slide` component
- if we update `Slide` and forget to update `AnimatedSlide`, the user will easily notice the difference (which would be bad!)

Reply:
Yeah, most of styles are identical, except for [this line](https://github.com/3perf/website/blob/d8ac74768b25a89cf46c949d896a229b2598fd06/src/components/talks/Slide/styled.ts#L9) and [this styled component](https://github.com/3perf/website/blob/d8ac74768b25a89cf46c949d896a229b2598fd06/src/components/talks/Slide/styled.ts#L126-L142). So, I guess we can make them shared.

Not sure, that making `AnimatedSlide` component extend `Slide` is the right thing to do, as it might create more problems, that it would solve. Sharing styled components should be enough.

Reply:
Utilized importing styles from `Slide` component to avoid repeating and avoid problems when changing styles.